.goreleaser.yml

```diff
@@ -0,0 +1,27 @@
+project_name: "argocd-vault-plugin"
+
+builds:
+  - binary: "argocd-vault-plugin"
+    flags: "-trimpath"
+    ldflags: >-
+      -s -w
+      -X "github.com/IBM/argocd-vault-plugin/version.Version={{.Tag}}"
+      -X "github.com/IBM/argocd-vault-plugin/version.BuildDate={{.Date}}"
+      -X "github.com/IBM/argocd-vault-plugin/version.CommitSHA={{.Commit}}"
+    env:
+      - "CGO_ENABLED=0"
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    tags:
+      - netgo
+      - static_build
+
+release:
+  prerelease: auto
```

Comment:
This should be set back to false, we dont normally do pre releases

Reply:
I saw some old release candidates so I thought it might be useful.  I think all this does is if you check the prerelease checkbox, it will still run the automation.  I don't think it does anything otherwise.